src/protagonist/DLCS.Repository/Strategy/SftpOriginStrategy.cs

```diff
@@ -51,8 +51,9 @@ public class SftpOriginStrategy : IOriginStrategy
         ConnectionInfo connectionInfo = GetConnectionInfo(originUri, port, basicCredentials);
 
         try
-        { 
-            var outputStream = await sftpReader.RetrieveFile(connectionInfo, originUri.AbsolutePath, cancellationToken);
+        {
+            var originPath = Uri.UnescapeDataString(originUri.AbsolutePath);

```

Comment:
this PR specifically talks about spaces, but `Uri.UnescapeDataString` feels like it should work with other escape character types - is this ok?